test/testcafe/spec-en-leaks/EnglishLeaks_spec.js

```diff
@@ -44,8 +44,6 @@ const ignoredMocks = [
   'authenticator-expired-password.json',
   'authenticator-expired-password-with-enrollment-authenticator.json',
   'authenticator-expired-password-no-complexity.json',
-  'authenticator-enroll-phone.json',
-  'authenticator-enroll-phone-voice.json',
   'authenticator-enroll-email.json',
   'authenticator-enroll-data-phone.json',
   'authenticator-enroll-data-phone-voice.json',
```

Comment:
These cannot be removed yet because we still have `+1` country code that's not translated. Please correct me otherwise, it doesn't make sense to translate the country code too. We'll have to re-implement how contrycode is retrieved if we intend to do this.

Reply:
yeah, I think that makes sense, but let's just check with PM once on that and try to ignore that section from the test suite by adding a class of `no-translate` to the html element. Refer to the [guideline](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2021917320/siw-localization-bug-fixing+guideline) for details.